win32ss/gdi/ntgdi/freetype.c

```diff
@@ -271,11 +271,11 @@ RemoveCacheEntries(FT_Face Face)
 
     ASSERT_FREETYPE_LOCK_HELD();
 
-    CurrentEntry = g_FontCacheListHead.Flink;
-    while (CurrentEntry != &g_FontCacheListHead)
+    for (CurrentEntry = g_FontCacheListHead.Flink;
+         CurrentEntry != &g_FontCacheListHead;
+         CurrentEntry = CurrentEntry->Flink)
```

Comment:
For this loop we cannot make this change, because the structure is freed inside `RemoveCachedEntry`.
To easier find such bugs, zero out (or fill with garbage) the memory, before freeing it.

Reply:
OK.